recipe/meta.yaml

```
@@ -19,7 +19,8 @@ build:
     - {{ PYTHON }} -m mesonbuild.mesonmain setup $MESON_ARGS builddir  # [not win]
     - {{ PYTHON }} -m mesonbuild.mesonmain setup %MESON_ARGS% builddir  # [win]
     - {{ PYTHON }} -m build --wheel --no-isolation --skip-dependency-check -Cbuilddir=builddir
-    - cd dist && {{ PYTHON }} -m pip install *.whl && cd ..
+    - {{ PYTHON }} -c "from glob import glob; import sys; sys.stdout.writelines(glob('dist/*.whl'))" > whls.txt
+    - {{ PYTHON }} -m pip install -r whls.txt
```

Windows dislikes the `*.whl` syntax and `pip` doesn't seem to handle globs well itself (so quoting didn't work). This just uses `glob` to write the wheel filename into a text file and install it that way

---

Sometimes I like the hoops we jump to make Windows work are not  worth it :-/

---

Saw a clever solution, which may help us to simplify this and still work on all platforms

```suggestion
    - {{ PYTHON }} -m pip install --find-links dist pywavelets
```